src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c

```diff
@@ -28,23 +28,155 @@ const ActorInit Obj_Entotu_InitVars = {
     (ActorFunc)ObjEntotu_Draw,
 };
 
-#endif
+u8 D_80A35410[][16] = {
+    { 0xFA, 0x7F, 0x14, 0x05, 0x01, 0x2D, 0x00, 0x00, 0xFF, 0x75, 0xF4, 0x69, 0x00, 0x00, 0x00, 0x00 },
+    { 0x00, 0x00, 0x08, 0xD5, 0x02, 0x5B, 0x00, 0x00, 0x02, 0x00, 0x03, 0xCC, 0x4D, 0x44, 0x34, 0xFF },
+    { 0x00, 0x00, 0x1B, 0xAC, 0x01, 0x2D, 0x00, 0x00, 0x02, 0x00, 0xE9, 0xD0, 0x00, 0x00, 0x00, 0x00 },
+    { 0xFB, 0xAD, 0x08, 0xD5, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00 },
+    { 0x05, 0x81, 0x14, 0x05, 0x01, 0x2D, 0x00, 0x00, 0x04, 0x8B, 0xF4, 0x69, 0x00, 0x00, 0x00, 0x00 },
+    { 0x04, 0x53, 0x08, 0xD5, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00 },
+    { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00, 0x10, 0x3A, 0x26, 0x21, 0x19, 0x7E },
+};
+
+s32 func_80A34700(s16 minutes) {
```

Comment:
This *might* be boolean, though I'm not completely convinced.

Reply:
It can return 0, 1, 2 or 3.

Reply:
Oh so it can, I missed the other settings of ret.